lib/base/novacom.js

```diff
@@ -72,7 +72,7 @@ const fs = require('fs'),
                 if (err) {
                     setImmediate(next, err);
                 } else {
-                    log.info("Resolver#load()", "devices:", resolver.devices);
+                    log.verbose("Resolver#load()", "devices:", resolver.devices);
```

Comment:
Do you want to print the log in verbose?

Reply:
Yes, If you want to keep it to "info" level, I'll change it to "info" as original.